MidasCivil_Adapter/Convert/ToBHoM/Properties/ToProfile.cs

```diff
@@ -45,19 +45,27 @@ public static IProfile ToProfile(List<string> sectionProfile, string shape, stri
                     double webSpacing = System.Convert.ToDouble(sectionProfile[4]).LengthToSI(lengthUnit);
                     double webThickness = System.Convert.ToDouble(sectionProfile[2]).LengthToSI(lengthUnit);
                     double corbel;
-                    if (System.Math.Abs(width / 2 - webSpacing / 2 - webThickness / 2) < oM.Geometry.Tolerance.Distance)
+                    if (webSpacing < oM.Geometry.Tolerance.Distance)
+                    {
+                        corbel = (width / 2 - webThickness / 2).LengthToSI(lengthUnit);
+                    }
+                    else if (System.Math.Abs(width / 2 - webSpacing / 2 - webThickness / 2) < oM.Geometry.Tolerance.Distance)
                     {
                         corbel = 0;
                     }
-
                     else
                     {
                         corbel = (width / 2 - webSpacing / 2 - webThickness / 2).LengthToSI(lengthUnit);
                     }
+                    double bottomFlangeThickness = System.Convert.ToDouble(sectionProfile[5]).LengthToSI(lengthUnit);
+                    if (bottomFlangeThickness < oM.Geometry.Tolerance.Distance)
+                    {
+                        bottomFlangeThickness = System.Convert.ToDouble(sectionProfile[3]).LengthToSI(lengthUnit);
+                    }
```

Comment:
See comment below about storing `topFlangeThickness` so can assign the value to `bottomFlangeThickness below.

Also, be consistent with `bottomFlangeThickness` - below you use botFlangeThickness`.